src/PIL/ImageCms.py

```diff
@@ -192,9 +192,9 @@ class ImageCmsTransform(Image.ImagePointHandler):
 
```

Comment:
In this file, should we also give code formatting/linking to:

* `ImageCms.buildTransform()` or `ImageCms.buildTransformFromOpenProfiles())`
* If you want to modify `im` in-place
* ImageCms.INTENT_PERCEPTUAL = 0 (DEFAULT)
  ImageCms.INTENT_RELATIVE_COLORIMETRIC = 1
  ImageCms.INTENT_SATURATION = 2
  ImageCms.INTENT_ABSOLUTE_COLORIMETRIC = 3
* CmsProfile


Should PyCMSError be code formatted rather than bold?


https://pillow--4682.org.readthedocs.build/en/4682/reference/ImageCms.html